MONEY/NetworkSpentSuvFastTravel.md

```diff
@@ -0,0 +1,20 @@
+---
+ns: MONEY 
+---
+
+## _NETWORK_SPENT_SUV_FAST_TRAVEL
+
+```c
+// 0x61A2DF64ED2D396E 
+void _NETWORK_SPENT_SUV_FAST_TRAVEL(int p0, BOOL p1, BOOL p2, Any p3);
+```
+
+```
+NativeDB Introduced: v2545
+```
+
+## Parameters
+* **p0**:
+* **p1**:
+* **p2**:
+* **p3**:
```

Comment:
Param's seems to be BOOL `amount`, BOOL `if from bank AND wallet` and CHAR `npcprovider`,